kinto/__main__.py

```diff
@@ -30,6 +30,8 @@ def main(args=None):
                         dest='backend',
                         required=False,
                         default=None)
+    parser.add_argument('--version', action='version', version=__version__,
```

Comment:
Should we also support the sort '-v' option?

    parser.add_argument('-v', '--version', action='version', ...